Cargo.toml

```diff
@@ -66,11 +64,13 @@ tokio-stream = { version = "0.1", features = ["net"] }
 
 [features]
 default = ["http-tls", "signing", "ws-tls-tokio", "ipc-tokio"]
-eip-1193 = ["js-sys", "wasm-bindgen", "wasm-bindgen-futures", "futures-timer/wasm-bindgen", "rand", "getrandom"]
-_http_base = ["hyper", "url", "base64", "headers"]
-http = ["_http_base", "hyper-proxy/tls"]
-http-tls = ["hyper-tls", "http"]
-http-rustls = ["hyper-rustls", "_http_base", "hyper-proxy/rustls-webpki"]
+wasm = ["js-sys", "wasm-bindgen", "wasm-bindgen-futures", "futures-timer/wasm-bindgen", "rand", "getrandom"]
+eip-1193 = ["wasm"]
+_http_base = ["reqwest", "bytes", "url", "base64", "headers"]
+http = ["_http_base"]
+http-tls = ["http", "reqwest/default-tls"]
+http-native-tls = ["http", "reqwest/native-tls"]
+http-rustls-tls = ["http", "reqwest/rustls-tls"]
```

Comment:
Would you mind adding two additional entries in `.github/workflows/rust.yml` to test these features work in isolation?

Reply:
I'm not familiar with Github action, tried and plz review.